packages/affiliates/scripts/example-balances.js

```diff
@@ -1,26 +1,27 @@
 const { BigQuery } = require("@google-cloud/bigquery");
 const Queries = require("../libs/bigquery");
-const moment = require("moment");
 const highland = require("highland");
 const { DecodeLog } = require("../libs/contracts");
-const { abi } = require("../../core/build/contracts/ExpiringMultiParty");
-const { EmpBalancesHistory } = require("../libs/processors");
-
-// uUSDwETH-DEC
-const empContract = "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56";
-const start = moment("2020-9-20", "YYYY-MM-DD").valueOf();
-const end = moment("2020-10-10", "YYYY-MM-DD").valueOf();
+const { getAbi } = require("@uma/core");
+const { EmpBalancesHistory, EmpBalances } = require("../libs/processors");
+
+const abi = getAbi("ExpiringMultiParty");
+// sanity check various contracts
+// const empContract = "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56";
+// const empContract = "0xaBBee9fC7a882499162323EEB7BF6614193312e3"
+// const empContract =   "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56"
```

Comment:
nit: did you mean to leave this comment here? If so, maybe we should label the different addresses for clarity?